handlers.py

```diff
@@ -18,16 +18,55 @@ def _send_JSON(server, source, JSON):
     data = json.dumps(JSON)
     server.send(data.encode('utf-8'), source)
 
+def _format_error_JSON(message):
+    print(message)
+    returnJSON = {
+      "type": MsgType.ERROR,
+      "message": message
+    }
+    return returnJSON
+
 def handle_movement(server, body, source):
     print('MOVEMENT') # TODO: Logging
     '''
-    TODO:
     1. Get desination IP from cache (if not in cache, get from 'cars' table)
     2. Forward the message
     '''
-    data = json.dumps(body).encode('utf-8')
-    car_ip = 'localhost' # TODO: Make this read from cache or 'cars' table
-    server.send(data, (car_ip, CAR_PORT))
+
+    # Get JSON data
+    car_name = body["car_name"]
+    userID = body["userID"]
```

Comment:
Our current design only has `x` and `y` fields for the MOVE messages. What's the reasoning for adding `car_name` and `userID`?  

Also, if we are identifying the car in the MOVE messages, shouldn't we be using the car's ID instead of its name?

Reply:
car_name and userID were added in order to identify the car. It was required when we changed the authentication process.

This JSON is received from the application. We can use the car's ID instead.

Reply:
Okay, then I'd say just change it to the car's ID and then merge.

Reply:
car_id used now.